### DISCUSSION TITLE
After Update to 11 - getObjectList error

### DISCUSSION THREAD

**Discussion by mediafant** [pimcore/pimcore#15844] [Q&A / Support] [answered]
Hi all,

After Update Page from 10 to 11.0.2 script with this line:
`$news = News::getList();`
end with this error-message:
`ucfirst(): Argument #1 ($string) must be of type string, null given`

And now I see in Pimcore Backend, there are no DataObjects.
When I rebuild them with:
`./bin/console pimcore:deployment:classes-rebuild`
.. it is full with Error like this:
`Column not found: 1054 Unknown column 'objects.id' in 'on clause'`

And if I login to Pimcore-Backend, I see this Error-Message:
`Message: An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'type' in 'field list'`

What is the reason, what can I do?

Andreas

**Comment by kingjia90** [comment, marked as answer]
`Column not found: 1054 Unknown column 'objects.id' in 'on clause'` looks like that the migration to change `objects.o_id` to `objects.id` didn't went well

**Reply by mediafant** [reply]
Thanks, that's it
